includes/class-volunteer.php

```diff
@@ -289,6 +289,11 @@ public function create_update_user( $form_fields ){
 			$userdata['ID'] = $existing_user;
 
 			$user_id = wp_update_user( $userdata );
+
+			//On multisite we need to add the user to this site if they don't have access
+			if( is_multisite() && !is_user_member_of_blog( $userdata['ID'] ) ){
+				add_user_to_blog( get_current_blog_id(), $userdata['ID'], 'volunteer' );
```

Comment:
This is slick - nice and straight forward. It's nice to see this in action and familiarize myself with some of these functions like the is_user_member_of_blog and add_user_to_blog calls.

Reply:
Definitely agree. Thanks for giving it a look.